commands/team.go

```diff
@@ -176,17 +177,26 @@ func archiveTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error
 }
 
 func listTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
-	teams, response := c.GetAllTeams("", 0, 10000)
-	if response.Error != nil {
-		return response.Error
-	}
+	page := 0
+	for {
+		teams, response := c.GetAllTeams("", page, web.LIMIT_MAXIMUM)
+		if response.Error != nil {
+			return response.Error
+		}
 
-	for _, team := range teams {
-		if team.DeleteAt > 0 {
-			printer.PrintT("{{.Name}} (archived)", team)
-		} else {
-			printer.PrintT("{{.Name}}", team)
+		for _, team := range teams {
+			if team.DeleteAt > 0 {
+				printer.PrintT("{{.Name}} (archived)", team)
+			} else {
+				printer.PrintT("{{.Name}}", team)
+			}
 		}
+
+		if len(teams) < web.LIMIT_MAXIMUM {
```

Comment:
Looks good. I double checked that teams endpoint doesn't error on empty page, otherwise if number of teams total was a multiple of `web.LIMIT_MAXIMUM` it would return error.  Instead, in that case, it will do one last request, get an empty list then exit clean.

Reply:
Maybe add a unit test for team count a multiple of `web.LIMIT_MAXIMUM` just in case the endpoint (e.g. v6) starts erroring for page numbers beyond the last one?  Not sure if necessary.

Reply:
Hum... it's an interesting corner case, but I think that if the API would start erroring on empty pages, we would treat those errors as bugs, so the test case wouldn't be needed. I can add it if you feel strongly about it tho